src/alt.js

```diff
@@ -77,7 +99,8 @@ class AltStore {
         )
       }
       if (model[LISTENERS][payload.action]) {
-        this[SET_STATE] = false
+        this[IS_DISPATCHING] = true
```

Comment:
Although if its in a lifecycle hook or another dispatch call then it loses its batching benefits...hmmm maybe there is a better way.